modules/rosa-create-objects.adoc

```diff
@@ -717,6 +717,9 @@ a|--cluster <cluster_name>\|<cluster_id>
 |--replicas
 |Required when autoscaling is not configured. The number (integer) of machines for this machine pool.
 
+|--tags                            
+|Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: 'key:value,foo:bar'
```

Comment:
The help message is like below
```
lixue@Xue-Lis-MacBook-Pro ~ % rosa create cluster -h|grep tags
      --tags strings                                          Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: 'key value, foo bar'
```

Reply:
Updated the text to:

![image](https://github.com/openshift/openshift-docs/assets/16167833/01772041-65a7-47a5-a356-87b5c8aef437)